include/dfu/suit_dfu_fetch_source.h

```diff
@@ -7,7 +7,20 @@
 #ifndef SUIT_DFU_FETCH_SOURCE_H__
 #define SUIT_DFU_FETCH_SOURCE_H__
 
-#if CONFIG_SUIT_STREAM_FETCH_SOURCE_MGR
+/** @file suit_dfu_fetch_source.h
+ *
+ * @brief The fetch source manager allows for registration of multiple data sources to be used by
+ *        the SUIT processor.
+ *
+ * @details
+ * The SUIT proccessor iterates over the registered fetch sources, calling respective
```

Comment:
Could you explain in this section what are "data sources" in SUIT? I guess I know, but I would prefer to be sure

Reply:
I meant in "brief" above you refer to "multiple data sources". I don't think this term was used before, so it would be good to explain it here in the "details" section.

Reply:
Changed to `multiple fetch sources` to avoid introducing multiple names for the same concept.